src/main/java/de/False/BuildersWand/NMS/v_1_8/v_1_8_R1.java

```diff
@@ -47,7 +47,7 @@ public boolean isMainHand(PlayerInteractEvent event)
     @Override
     public String getDefaultParticle()
     {
-        return Effect.COLOURED_DUST.toString();
+        return Effect.valueOf("COLOURED_DUST").toString();
```

Comment:
Those weird things in the 1.8 NMS classes are needed to compile the plugin successfully (Sorry for that, looks awful, I know... But I didn't have another choice)